t/22-dashboard.t

```diff
@@ -501,6 +501,11 @@ subtest 'proper build sorting for dotted build number' => sub {
     $group->update({build_version_sort => 0});
     @build_names = reverse @build_names;
     check_builds(\@build_names, $group, 'builds shown sorted by dotted number');
+
+    # sorted by oldest job per build (which is the same here as there is no overlap)
+    $group->update({build_version_sort => 2});
+    check_builds(\@build_names, $group, 'builds shown sorted by dotted number');
```

Comment:
It is not nice to repeat the same check-description (even though this is already the case with the previous tests).

It is also looks wrong that the expected `@build_names` are the same as for the previous case.